screen/Menu/Menu.screen.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { View, Text, TouchableHighlight } from 'react-native';
+import styles from "./Menu.style"
+import colors from '../../res/colors';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+
+
+const Menu = ({ navigation }) => {
+    return (
+        <View >
+            <TouchableHighlight
```

Comment:
how about to create folder 'components' on the same level as 'nav' and 'res' and 'screen' folders and create inside of if it component Button that will pass props like: style, underlayColor, onPress, children and use it here? 
Some help: each common component should be created inside of folder with the same name. Folder 'components' should have also index.js file.